docs/index.rst

```diff
@@ -0,0 +1,47 @@
+.. nuclio documentation master file, created by
+   sphinx-quickstart on Wed Sep 20 10:10:53 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Nuclio documentation!
+==================================
+
```

Comment:
Learn how to use Nuclio to quickly develop "serverless" applications.

To download Nuclio and view full product release notes, visit the Nuclio GitHub [Releases](https://github.com/nuclio/nuclio/releases) page.

For support and additional product information, join the active [Nuclio Slack](https://nuclio-io.slack.com/) workspace.

(Header) Table of Contents: